spec/models/stat_spec.rb

```diff
@@ -53,4 +53,80 @@
       expect(stats.rdvs_group_by_type[["sur place", "05/04/2020"]]).to eq(1)
     end
   end
+
+  describe "#users_group_by_week" do
+    it "returns user count group by created at" do
+      now = Time.zone.parse("20220123 13:00")
+      travel_to(now)
+      create(:user)
+      stats = described_class.new(users: User.all)
+      expect(stats.users_group_by_week).to eq({ now.strftime("%d/%m/%Y") => 1 })
```

Comment:
Est-ce que ça serait pas plus clair avec la date formatée explicitement dans le résultat attendu?

```suggestion
      expect(stats.users_group_by_week).to eq({ "23/01/2022" => 1 })
```